dockers/docker-namespace/Dockerfile.j2

```diff
@@ -0,0 +1,2 @@
+FROM debian:stretch
```

Comment:
why do we need this docker?

Reply:
we can use the database as the namespace container, all other containers will use the same namespace as the database container. we do not need a separate namespace container.

Reply:
I have this since I start the namespaces, then the topology service is started, then the other containers start. I wanted to have the topology set up before any SONiC containers were started.

Reply:
topology service is only required for kvm image. it is not necessary to have topology build before all containers start. We can build topology after database docker started. in fact, the bgp session will not be able to setup until host interfaces are created.

the only hard requirement I see is that the peer links has to be start before the syncd docker starts. since syncd will link the peer device  with the host interface.